application/F3DStarter.cxx

```diff
@@ -713,12 +713,16 @@ int F3DStarter::Start(int argc, char** argv)
   // but this duplicate the initialization value as it is present in
   // F3DOptionTools::DefaultAppOptions too
   bool dryRun = false;
+  if (cliOptionsDict.find("dry-run") != cliOptionsDict.end())
+  {
+    dryRun = f3d::options::parse<bool>(cliOptionsDict["dry-run"]);
+  }
   if (cliOptionsDict.find("no-render") != cliOptionsDict.end())
   {
     dryRun = f3d::options::parse<bool>(cliOptionsDict["no-render"]);
```

Comment:
this looks wrong ?

Reply:
> this looks wrong ?

why?

Reply:
the `no-render` option should not impact the `dryRun` boolean. I think there is a bug in this code